### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by pascualin**
This PR only solves the following issues:

- Make Cyclops pages show the default menu
- Make Cyclops Navigation Bar 64px height
- Fix issue with CenturyLink logo on the top right part of the page

More fixes on the next PR

### CODE REVIEW

**Comment by jsmecham** [comment]
Not sure if you intended this to be reviewed yet, but just noticed that it causes overlapping with the main menu:

screenshot

This can be fixed by updating https://github.com/CenturyLinkCloud/Cyclops/blob/master/src/scripts/models/mainNavViewModel.coffee#L94. It's ugly hardcoding there (also what my flexbox branch was supposed to fix).

**Comment by pascualin** [comment]
Good call, looking at it now,

Thanks!

Ignacio Martín Prieto
Software Engineer
Plaza de Colon 2, Torre 1, Planta 5. Madrid 28046. Spain.
Office: +34 912922999<tel:+34%20912%2092%2029%2099> / Mobile: +34 654780610<tel:+34%20654%2078%2006%2010>
Email: ignacio.martin@ctl.io<mailto:ignacio@elasticbox.com>

On Apr 26, 2017, 9:46 PM +0200, Justin Mecham <notifications@github.com>, wrote:

Not sure if you intended this to be reviewed yet, but just noticed that it causes overlapping with the main menu:

[screenshot]<https://cloud.githubusercontent.com/assets/4340/25453541/537db8b2-2a97-11e7-9fc1-a37aa7762a62.png>

This can be fixed by updating https://github.com/CenturyLinkCloud/Cyclops/blob/master/src/scripts/models/mainNavViewModel.coffee#L94. It's ugly hardcoding there (also what my flexbox branch was supposed to fix).

**Comment by pascualin** [comment]
@mattnickles Please review

**Comment by jsmecham** [comment]
I've addressed your comments, I believe. The things under "Few things missing." are addressed in a separate PR at https://github.com/CenturyLinkCloud/dt-GlobalNavigationMenu/pull/4.

File: src/less/servicesMenu.less [hunk 1]
```diff
@@ -22,51 +21,73 @@
...
         text-decoration: none;
-        border-radius: 5px;
-        padding: 3px 8px 8px 8px;
-        margin-top: 3px;
+        border-radius: 3px;
+        padding: 3px 8px 7px 8px;
+        align-self: center;
+        border: 1px solid transparent;
+
+        &:hover {
```
Comment (mattnickles) [inline src/less/servicesMenu.less]: Hover state shouldn't be visible when active/open

File: src/less/servicesMenu.less [hunk 2]
```diff
@@ -22,51 +21,73 @@
...
-        font-family: "Open Sans", "Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif;
         display: flex;
         flex-direction: column;
         height: 45px;
         text-decoration: none;
-        border-radius: 5px;
-        padding: 3px 8px 8px 8px;
-        margin-top: 3px;
+        border-radius: 3px;
+        padding: 3px 8px 7px 8px;
```
Comment (mattnickles) [inline src/less/servicesMenu.less]: @jsmecham Can we adjust the padding here or on the arrow to ensure both sides are 10px?
image

**Comment by pascualin** [comment]
Ey @mattnickles. Could you please review the latest changes on this PR. This fixes the issues with the Open Sans font on Cyclops among many other issues.

File: src/less/nav.less [hunk 3]
```diff
@@ -245,3 +236,25 @@
 }
 
 }
+
+global-menu-container {
```
Comment (mattnickles) [inline src/less/nav.less]: @pascualin what's the purpose of this? the container is overlaying the whole screen and making all links, buttons, etc. inaccessible.

image

Comment (pascualin) [inline src/less/nav.less]: @mattnickles thats the Global Menu container, where the list of links to Managment, Public Cloud, etc.. go. It should be hidden at that point (if the menu is not open)

Comment (mattnickles) [inline src/less/nav.less]: @pascualin can you investigate why it's displaying on all.html? also, the position appears to be broken on each of the example pages. the backdrop isn't fixed and it's scrolling with the underlying page content.
image

**Comment by pascualin** [comment]
@mattnickles The two issues have now been solved. Please check.

**Pull request merged**